source/type/type.cpp

```diff
@@ -752,6 +752,9 @@ std::string static_data::captionByType(Type_e type){
 }
 
 Mode_e static_data::modeByCaption(const std::string & mode_str) {
+    if (mode_str == "ridesharing") {
```

Comment:
It might be smarter to test that it's one of `['walking', 'car', 'bss', 'bike']`
Any opinion @xlqian @kinnou02 @TeXitoi ?

Reply:
It is, we won't add new modes in kraken